main.go

```diff
@@ -261,12 +261,12 @@ func main() {
 
 	// +kubebuilder:scaffold:builder
 
-	if err := mgr.AddReadyzCheck("ping", healthz.Ping); err != nil {
+	if err := mgr.AddReadyzCheck("webhook", mgr.GetWebhookServer().StartedChecker()); err != nil {
 		setupLog.Error(err, "unable to create ready check")
 		os.Exit(1)
 	}
 
-	if err := mgr.AddHealthzCheck("ping", healthz.Ping); err != nil {
+	if err := mgr.AddReadyzCheck("webhook", mgr.GetWebhookServer().StartedChecker()); err != nil {
```

Comment:
```suggestion
	if err := mgr.AddHealthzCheck("webhook", mgr.GetWebhookServer().StartedChecker()); err != nil {
```

Reply:
@sbueringer Thanks for the review. But unfortunately closing it as @sedefsavas was already working on it.

Reply:
@sbueringer Updated the suggested change. Please take a look. Thanks in advance.

Reply:
@geetikabatra Looks good thx. I think you have to change your commit message: https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/2685#issuecomment-902636137 (and drop the link to the issue)

Reply:
Done!